### PR TITLE
Simplify Scope discussion.

### LINKED ISSUE #8: The scope section is probably too verbose
I wonder how to make the scope section much more succinct. I am not sure that the detailed bulleted list for the changes should be part of the charter. Let alone the fact that it sounds as if those entries, and their proposed solutions (referred to via the PR-s) are already cast in concrete and the charter takes a commitment. Formally, it will be up to the WG to make those decisions.

Is there a text/blog/presentation (like, e.g., the TPAC presentation) that could be referred to instead, showing the type of solution adopted at this moment by the CG? That could be _referred_ from the charter but not turning these into a WG commitments? 

Cc: @wseltzer

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes #8.

### CODE REVIEW

**Comment by gkellogg** [comment]
I think this could be better, and welcome help in improving the wording.

File: index.html [hunk 1]
```diff
@@ -231,8 +229,8 @@ <h3 id="out-of-scope">Out of Scope</h3>
             <p>The following features are out of scope, and will not be addressed by this Working Group.</p>
 
             <ul class="out-of-scope">
-              <li class="todo">There must be something...</li>
-            </ul>
+              <li>There must be something...</li>
+     ∑       </ul>
```
Comment (davidlehn) [inline index.html]: Still a "todo" and typo char added.

Comment (gkellogg) [reply index.html]: Out-of-scope addressed in #10.

File: index.html [hunk 2]
```diff
@@ -205,20 +205,18 @@ <h2>Scope</h2>
...
+        <p>The <a href="" class="todo">JSON-LD Working Group</a> will work to address the needs
+          of the community as expressed through <a href="https://lists.w3.org/Archives/Public/public-linked-json/">email</a> and <a href="https://github.com/json-ld/json-ld.org/issues?utf8=✓&q=is%3Aissue+created%3A%3E2014-01-01+-label%3Awebsite+-label%3Aplayground+">issues</a> along with other
+          emergent considerations of stakeholders for expressing data using JSON
+          and related data formats that are consistent with the general objectives
+          for describing data on the World Wide Web as expressed in
+          <a href="https://www.w3.org/TR/dwbp/">Data on the Web Best Practices</a>.
+          The group will further the Application Programming Interface to include
+          Framing, Filtering and other transformation interfaces.</p>
+        <p>A summary of changes addressed by the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a>
```
Comment (iherman) [inline index.html]: I would rather say "proposed" by the CG (for the purposes of this charter). It makes the relationship between the two groups cleaner.

**Comment by iherman** [comment]
I think, in line with the [remarks](https://github.com/json-ld/charter/issues/8#issuecomment-360503288) of @wseltzer, that the two references to the JSON LD usage (data commons and W3Tech) could be removed and the two adjacent paragraphs merged. Instead, the two sets of information should be put somewhere else and referred to from this charter.

The "somewhere" else can be a separate Wiki page in the same repo, for example. Actually, it is also all right if the removed, detailed bullet list with he changes that have been removed is also put into such a wiki page and referred from the charter.

**Comment by iherman** [comment]
(Admin: let us not forget to close #8 when we merge this:-)

**Comment by BigBlueHat** [comment]
@iherman #8 will actually auto close if/when this PR is merged--because GitHub 🤖 understands "fix" or "fixes" + issue number. If you mouse-over "Fixes" in @gkellogg's original comment, you'll see what I mean. 😸

**Comment by iherman** [comment]
I knew about that but I forgot:-(

> @iherman <https://github.com/iherman> #8 <https://github.com/json-ld/charter/issues/8> will actually auto close if/when this PR is merged--because GitHub 🤖 understands "fix" or "fixes" + issue number. If you mouse-over "Fixes" in @gkellogg <https://github.com/gkellogg>'s original comment, you'll see what I mean. 😸
>

**Comment by azaroth42** [comment]
Rather than leave detailed comments as to changes, I made a PR to move some text around: 
 https://github.com/json-ld/charter/pull/12

**Comment by gkellogg** [comment]
I started on a [wiki](https://github.com/json-ld/json-ld.org/wiki/Changes-in-Community-Group-Drafts-Targeted-for-1.1). I wanted to put stuff in from the TPAC presentation, too. @danbri’s been asking for something anyway, and it can serve dual purpose.

**Comment by danbri** [comment]
Thanks Gregg! 

As a well-wisher who isn't deeply immersed in this group's discussion, it would massively help to have actual markup examples to illustrate each. Eg. the named graph mention was intriguing but I don't understand it. It would be nice if json-ld++ had some idiom for annotating edges in the extracted graph with simple key value pairs, even if those were dropped in the basic rdf mapping. Is that too ambitious an idea for this work?

BTW I spent time this week with a variety of colleagues who work with structured data markup (json-ld, microdata and rdfa), and learned a bit more of some of the practical issues they encounter. One is around ordering, and the fact that each syntax has its own noration for lists, plus Schema.org's own ItemList abstraction also in the mix. In a JSON-LD context people easily read any/all use of [ and ] as implying that order is being explicitly represented and then are surprised when additional verbosity is required. While the ancient history of the JSON spec being  ambiguous  w.r.t. repeated fields, combined with RDF's unordered data model explains this, it doesn't make the markup any more attractive.

The other thing that comes up in a json-ld-in-html setting is the desire to avoid (like rdfa/microdata) content duplication by pointing out with references to portions of the enclosing DOM to pull in chunks of markup into the extracted representation. We have made some experiments in that direction  but would prefer to have something standard.

**Comment by azaroth42** [comment]
Hi @danbri! I think both points warrant discussion, and we should create issues on the regular JSON-LD repository to track them. Would you like to, or I'm happy to cut and paste?

For the order question, I recall this discussion: https://lists.w3.org/Archives/Public/public-linked-json/2014Apr/0007.html that might also serve as a useful starting point.  (Simply replace OLO in the message, with ItemList, and I think the point is the same)

**Comment by gkellogg** [comment]
> It would be nice if json-ld++ had some idiom for annotating edges in the extracted graph with simple key value pairs, even if those were dropped in the basic rdf mapping. Is that too ambitious an idea for this work?

Given the underlying RDF data model, annotating individual edges is problematic. But, we could imagine having some kind of support for `schema:Role`, which is a kind of reification. Not sure what that would look like.

Of course, introducing named graphs was to allow them to be annotated, and you can always create a named graph with triples in it, and use triples in the default graph to talk about it; Verifiable Claims essentially do something like this.

> In a JSON-LD context people easily read any/all use of [ and ] as implying that order is being explicitly represented and then are surprised when additional verbosity is required. 

Sure, it's a constant point of confusion, but again, RDF data model ... At least, JSON-LD makes lists simpler by using `@container: @list` in the context term definition, which solves most practical uses.

As @azaroth42 pointed out, we did conceive of other ordering mechanisms, such as [The Ordered List Ontology](http://smiy.sourceforge.net/olo/spec/orderedlistontology.html), I think we could certainly do something for ItemList (`@itemlist`?). Filing an issue would be a great place to start.

> avoid (like rdfa/microdata) content duplication by pointing out with references to portions of the enclosing DOM

I'd like to see what you're thinking here. In my view, the graph created by an HTML page is the union of the different markups contained on the page (RDFa, microdata and JSON-LD). From a data-model perspective, the JSON-LD could certainly reference resources that are defined through other markup, or in another JSON-LD blob. CSVW would be great to through into the mix, too.

**Comment by gkellogg** [comment]
Obviated by #12.

**Pull request closed without merging**